code/__DEFINES/dcs/signals/signals_object.dm

```diff
@@ -332,6 +332,9 @@
 ///called in /obj/item/gun/fire_gun (user, target, flag, params)
 #define COMSIG_GUN_TRY_FIRE "gun_try_fire"
 	#define COMPONENT_CANCEL_GUN_FIRE (1<<0)
+///called in /obj/item/gun/try_fire_gun (user, src, target, params)
+#define COMSIG_MOB_TRYING_TO_FIRE_GUN "mob_trying_to_fire_gun"
+	#define COMPONENT_CANCEL_FIRING_GUN (1<<0)
```

Comment:
Should just use the same return as above. `COMPONENT_CANCEL_GUN_FIRE`

Reply:
👍